### ISSUE TITLE
Problem: tests mutate source folder

### ISSUE THREAD

**Issue by sevein** [artefactual/archivematica-storage-service#220]
Tests should generate temporary data somewhere else.

How to reproduce:

```
$ docker run \
    -e DJANGO_SETTINGS_MODULE=storage_service.settings.test \
    --entrypoint pytest -t ss

administration/tests/test_languages.py ....
locations/tests/test_api.py ..............F..F...F.......
locations/tests/test_arkivum.py FFFFFF
locations/tests/test_dataverse.py ...F
locations/tests/test_dspace.py ..F..F.FF
locations/tests/test_duracloud.py .......FFFFFFFFFF
locations/tests/test_fixity_log.py .
locations/tests/test_locations.py ...
locations/tests/test_lockssomatic.py .
locations/tests/test_package.py ..........
locations/tests/test_swift.py .....FFFF.
storage_service/tests/test_shibboleth.py sssss
storage_service/tests/test_startup.py ....
```

Running as root fixes the problem but it's not ideal:

```
$ docker run \
    --user=root \
    -e DJANGO_SETTINGS_MODULE=storage_service.settings.test \
    --entrypoint pytest -t ss

administration/tests/test_languages.py ....
locations/tests/test_api.py .............................
locations/tests/test_arkivum.py ......
locations/tests/test_dataverse.py ....
locations/tests/test_dspace.py .........
locations/tests/test_duracloud.py .................
locations/tests/test_fixity_log.py .
locations/tests/test_locations.py ...
locations/tests/test_lockssomatic.py .
locations/tests/test_package.py ..........
locations/tests/test_swift.py ..........
storage_service/tests/test_shibboleth.py sssss
storage_service/tests/test_startup.py ....
```

**Comment by sevein** [comment]
Why this is not a problem in Travis CI?

**Comment by sevein** [comment]
Now I remember, this is not a problem in Travis CI likely because the working tree is owned by the same user that running the tests. This is not true in our Docker image where the files are likely owned by root but the user running the tests isn't.

**Comment by sevein** [comment]
This was fixed in #276.
